src/main/java/it/gov/pagopa/idpay/transactions/controller/TransactionsControllerImpl.java

```diff
@@ -0,0 +1,23 @@
+package it.gov.pagopa.idpay.transactions.controller;
+
+import it.gov.pagopa.idpay.transactions.model.RewardTransaction;
+import it.gov.pagopa.idpay.transactions.service.RewardTransactionService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Flux;
+
+@RestController
+@Slf4j
+public class TransactionsControllerImpl implements TransactionsController{
+    private final RewardTransactionService rewardTransactionService;
+
+    public TransactionsControllerImpl(RewardTransactionService rewardTransactionService) {
+        this.rewardTransactionService = rewardTransactionService;
+    }
+
+    @Override
+    public ResponseEntity<Flux<RewardTransaction>> findAll(String idTrxAcquirer, String userId, String trxDate, String amount) {
+        return ResponseEntity.ok(rewardTransactionService.findTrxsFilters(idTrxAcquirer, userId, trxDate, amount));
```

Comment:
The errors to return if mandatory filters are not met should be thrown here, because it's a limit that we are applying just to API